Enable C language. Fix configure on Arch at:

-- FreeImage test failed to compile - This may indicate a build system bug
CMake Error at /usr/lib64/cmake/ignition-cmake0/cmake0/FindFreeImage.cmake:46 (try_run):
Unknown extension ".c" for file

/home/nim/local/robotpkg/robotpkg/wip/talos-data/work.Nausicaa/talos_data-1.1.0/CMakeTmp/test_freeimage.c

try_compile() works only for enabled languages.

--- CMakeLists.txt.orig	2019-12-18 10:44:33.000000000 +0100
+++ CMakeLists.txt	2020-01-25 21:44:05.988941613 +0100
@@ -13,7 +13,7 @@
 INCLUDE(cmake/base.cmake)
 
 # Specify the project.
-COMPUTE_PROJECT_ARGS(PROJECT_ARGS LANGUAGES CXX)
+COMPUTE_PROJECT_ARGS(PROJECT_ARGS LANGUAGES CXX C)
 PROJECT(${PROJECT_NAME} ${PROJECT_ARGS})
 
 find_package(catkin REQUIRED COMPONENTS
